ansible-ipi-install/roles/shared-labs-prep/tasks/main.yml

```diff
@@ -146,6 +146,20 @@
   tags:
     - clearjobs
 
+- name: Wait for iDrac to be responsive (check via --check-boot)
+  shell:
+    chdir: "{{ badfish_tempdir.path }}/src/badfish"
+    cmd: |
+      source {{ badfish_venv }}/bin/activate
+      {{ badfish_cmd }}{{ item }} --check-boot
+  with_items:
+    - "{{ master_fqdns }}"
+    - "{{ worker_fqdns }}"
+  register: wait_for_idrac
+  until: wait_for_idrac is succeeded
+  retries: 5
```

Comment:
That's only 2.5 minutes; I've certainly seen it take 5 minutes or more.  I'd suggest waiting for about 10 minutes.

Reply:
I think the playbook has other problems too; clearing the job queue shouldn't retry, it should just wait for --check-boot to succeed.  Setting nodes to director boot order probably should simply wait until that happens, rather than retrying the command.  But I'll give this a try (with a 10 minute timeout) first.